### ISSUE TITLE
Replace instances of .to(unit).value with .to_value(unit)

### ISSUE THREAD

**Issue by adrn** [astropy/astropy#6136]
Once #6127 is merged, we should replace any instances of `.to(unit).value` in code or documentation to `.to_value(unit)` as it will be faster.

**Comment by adrn** [comment]
Hm, I labeled this `units` but actually it touches many of the subpackages because it is a large find-and-replace...

**Comment by pllim** [comment]
Okay, I removed units and added docs.

**Comment by adrn** [comment]
Ok -- it applies mostly to code, but there are a few instances in the docs as well.

**Comment by mhvk** [comment]
Oops, I had already done this in the other PR. Will make a new one just for the replacement...

**Comment by mhvk** [comment]
See #6137
